actionview/lib/action_view/helpers/form_tag_helper.rb

```diff
@@ -134,13 +134,15 @@ def select_tag(name, option_tags = nil, options = {})
 
         if options.include?(:include_blank)
           include_blank = options.delete(:include_blank)
+          options_for_blank_options_tag = { value: '' }
 
           if include_blank == true
             include_blank = ''
+            options_for_blank_options_tag[:label] = ' '
```

Comment:
It needs to be any Text that is not `inter-element whitespace`, so no empty string- `""`
Tried to keep minimal with just a space.
Ref: https://www.w3.org/TR/html5/forms.html#the-option-element